src/Kinepolis/KinepolisService.php

```diff
@@ -144,6 +149,16 @@ private function process(ParsedMovie $parsedMovie, string $token): void
             $eventId = $this->createNewMovie($parsedMovie);
             $addImage = $this->uploadImage($token, $parsedMovie, $eventId);
             $commands[] = $addImage;
+
+            $trailer = $this->trailerRepository->search($parsedMovie->getTitle()->toString());
```

Comment:
I wonder if we need to add some logging about the retrieved trailer and maybe a catch statement.
I also wonder if we need to add a catch statement to continue the import even if the trailer search is failing.